src/nvim/syntax.c

```diff
@@ -6826,7 +6826,7 @@ int hl_combine_attr(int char_attr, int prim_attr)
     // Copy all attributes from char_aep to the new entry
     new_en = *char_aep;
   } else {
-    memset(&new_en, 0, sizeof(new_en));
+    new_en = (attrentry_T)ATTRENTRY_INIT;
```

Comment:
is this not redundant, as `new_en` already has the correct value?

Reply:
seems redundant indeed